tests/test_wordpress_release_pipeline.py

```
@@ -0,0 +1,396 @@
+    assert "opcache-reset=skipped" in result.stdout
+
+
+def test_code_sync_never_ships_gitignored_files():
+    deploy = (ROOT / "wordpress/scripts/deploy-wp-code.sh").read_text()
+
+    assert "--filter=:- .gitignore" in deploy
+    assert "--exclude=.env" in deploy
+
+
+def test_release_uses_one_timestamp_for_log_backup_and_marker(tmp_path):
+    commit, env, events = _fixture(tmp_path)
```

Ruff (0.16.1)

[warning] Unpacked variable `events` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)